docs/install.rst

```diff
@@ -90,6 +93,11 @@ Disable and remove the swapfile once the virtual enviroment is constructed::
   user@webserver:~$ sudo swapoff /tmp/swapfile
   user@webserver:~$ sudo rm /tmp/swapfile
 
+Custom Python environment
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note that if you have configured any custom Python environment variables on your system, make sure they are properly (un)set when running `certbot-auto` and `sudo`. This could happen if you run `sudo zsh` as your user's `PYTHONPATH` will be preserved which might cause problems.
```

Comment:
It might be nice to link to make "Python environment variables" link to https://docs.python.org/2/using/cmdline.html#environment-variables. You can do this like:

```
...custom `Python environment variables <https://docs.python.org/2/using/cmdline.html#environment-variables>`_ on..
```

Also, please correct me if I'm wrong, but I don't think it's the default behavior of either `sudo` or `zsh` to preserve environment variables like `PYTHONPATH`. If this is the case, I'd remove the second sentence entirely. You can add "(such as `PYTHONPATH`)" to the first sentence if you want to bring more attention to this variable.